recipes-core/images/mistysom-image.bb

```diff
@@ -7,7 +7,7 @@ LICENSE = "MIT"
 inherit core-image features_check
 inherit extrausers
 
-DISTRO_FEATURES = "x11 opengl"
+REQUIRED_DISTRO_FEATURES = "x11 opengl"
```

Comment:
as previously mentioned, may need to be updated to `Wayland` for hardware acceleration:
As for why Renesas ussed Wayland rather than X11 in the BSP, Chris explains:
> The technical reason is that the GPU libraries support OpenGL ES API. This API is better suited for Wayland than X11.
> So, you could get an X11 system to run, but there would be no GPU acceleration.
> Also, the Wayland/Weston implementation is much smaller and better suited for embedded systems. For X11 is fine for PCs with big processors and lots of memory, Wayland/Weston is a much better choice for a SoC bases platform.

Reply:
@matinlotfali please review the above